deploy/Dockerfile-slim

```
@@ -33,23 +33,20 @@ ADD Makefile /src
 RUN yarn install --force
 ADD .snyk /src
 ENV SNYK_TOKEN=${snyk_token}
-RUN npm run snyk-protect
+RUN npm run snyk-protect || true
```

```suggestion
RUN npm run snyk-protect
```
this was useful for local dev